bitmind/validator/miner_performance_tracker.py

```diff
@@ -0,0 +1,115 @@
+from sklearn.metrics import accuracy_score, precision_score, recall_score, f1_score, matthews_corrcoef, roc_auc_score
```

Comment:
Should this be in the `validator/` directory, or somewhere else? Maybe a directory dedicated to miners?

Edit: Nevermind, I misunderstood the usage of this